wbooks/src/redux/book/actions.ts

```diff
@@ -1,10 +1,20 @@
 import { createTypes, completeTypes } from 'redux-recompose';
 import { getBooks } from '@services/BookService';
 
-export const actions = createTypes(completeTypes(['GET_BOOKS']), '@@BOOKS');
+export const actions = createTypes(completeTypes(['GET_BOOKS'], ['FILTER_BOOKS']), '@@BOOKS');
 
 export const actionCreators = {
-  getBooks: () => ({ type: actions.GET_BOOKS, target: 'books', service: getBooks })
+  getBooks: () => ({
+    type: actions.GET_BOOKS,
+    target: 'books',
+    service: () => getBooks()
```

Comment:
Why this?

Reply:
I don't understand what's wrong with this

Reply:
Why the change? Keep it like: `service: getBook`, don't create a new function for nothing

Reply:
Ahhh, now i understand. I did that because i tried to do the search with that component, evidently it didn't worked out as expected and forgot to revert the change.